sound/hybrid_font.h

```diff
@@ -182,7 +186,12 @@ class HybridFont : public SaberBase {
 	e = &drag;
       }
       if (!lock_player_) {
-	lock_player_ = PlayPolyphonic(e);
+        if (bgnlock.files_found()) {
+          lock_player_ = PlayPolyphonic(&bgnlock);
+        } else {
+	  lock_player_ = PlayPolyphonic(e);
```

Comment:
indentation is off

Reply:
Not sure how that happened but oops! Lemme fix.